src/main/java/IllegalTimeFormatException.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Encapsulate the exception thrown when user time input is not recognised.
+ */
+public class IllegalTimeFormatException extends DukeException {
+
+    /**
+     * Constructs the IllegalTimeFormatException object.
+     *
+     * @param msg Message of the exception.
+     */
+    public IllegalTimeFormatException(String msg) {
+        super(msg);
+    }
+}
```

Comment:
Nice, i think it is a good idea to specify more exception variety like u did, it really helps readability :)